src/network_analysis.jl

```diff
@@ -542,6 +542,23 @@ end
 
 ### Conservation Laws ###
 
+# Implements the `conservationquantity` parameter metadata.
+struct ConservationQuantity end
+Symbolics.option_to_metadata_type(::Val{:conservationquantity}) = ConservationQuantity
```

Comment:
```suggestion
Symbolics.option_to_metadata_type(::Val{:conserved}) = ConservationQuantity
```
Just doing `@parameter A [conserved = true]` seems cleaner and clear?